samtranslator/model/appsync.py

```diff
@@ -23,3 +25,18 @@ class GraphQLApi(Resource):
     AuthenticationType: str
     Tags: Optional[List[Dict[str, Any]]]
     XrayEnabled: Optional[bool]
+
+    runtime_attrs = {"api_id": lambda self: fnGetAtt(self.logical_id, "ApiId")}
+
+
+class GraphQLSchema(Resource):
+    resource_type = "AWS::AppSync::GraphQLSchema"
+    property_types = {
+        "ApiId": PropertyType(True, IS_DICT),
```

Comment:
Why is it `IS_DICT` on top but `str` below?

Reply:
The `Intrinsicable` type is a dictionary and then that inside type is what the specific values type is. `ApiId` is a string, but it's wrapped in a `GetAtt` intrinsic.